core/core_bind.cpp

```diff
@@ -1192,11 +1188,11 @@ Vector<uint8_t> File::get_buffer(int64_t p_length) const {
 }
 
 String File::get_as_text() const {
-	ERR_FAIL_COND_V_MSG(!f, String(), "File must be opened before use, or is lacking read-write permission.");
+	ERR_FAIL_COND_V_MSG(f.is_null(), String(), "File must be opened before use, or is lacking read-write permission.");
 
 	String text;
 	uint64_t original_pos = f->get_position();
-	f->seek(0);
+	const_cast<FileAccess *>(*f)->seek(0);
```

Comment:
Why is the const cast needed here?

Reply:
`seek` which is not `cosnt` (that seems reasonable) is called from `const` `get_as_text` method (which restores the position, so I guess it's also reasonable for it to be `const`).

Before this PR, not changing the pointer to `FileAccess` was enough, but with `Ref<>` compiler checking if called method's `const` qualifier matches as well.